content/5_Plat/Sweep_Line.mdx

```diff
@@ -67,6 +67,90 @@ Extending Solution 1, we can use a set instead of divide and conquer. Once again
 As we visit point $P$, we utilize the set to consider all points with y-coordiate in $[P_y-\delta, P_y+\delta]$. The set contains $[P_x-\delta, P_x]$ because of how it is maintained as a running window. Now we have the same bounding box as Solution 1, with at most 6 points inside.
 
 For each point, we recalculate $\delta = \min(\delta, \delta_p)$, and update our set accordingly. Each point is inserted and removed from the set at most once, so the algorithm yields $\mathcal{O}(n\log n)$.
+
+<LanguageSection>
+
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+struct Point {
+	long double x, y;
+	bool operator<(const Point &other) {
+		if (x == other.x) return y < other.y;
+		return x < other.x;
+	}
+};
+
+const pair<Point, Point> INF = {{-1e9, -1e9}, {1e9, 1e9}};
+long double dist(pair<Point, Point> a) {
+	long double d1 = a.first.x - a.second.x;
+	long double d2 = a.first.y - a.second.y;
+	return sqrt(d1 * d1 + d2 * d2);
+}
+
+pair<Point, Point> min(pair<Point, Point> a, pair<Point, Point> b) {
```

Comment:
Overloading the `std::min` function like this is confusing. Use a more descriptive name for this function.

Reply:
I changed it to `get_closest_points` .